package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Artists8",
+  "version": "1.0.0",
+  "description": "## Description:",
+  "dependencies": {
```

Comment:
Where are express and express-handlebars? These need to be added.

Reply:
I think this is a problem with your version of node/npm, you may have to update them.